NorsokSCDLibrary.aml

```diff
@@ -25548,6 +25573,51 @@
         </SystemUnitClass>
       </SystemUnitClass>
     </SystemUnitClass>
+    <SystemUnitClass Name="VendorElementaryFunctionClass" ID="9dd97385-6013-45d2-bf94-9968144d7af8">
```

Comment:
Are we talking about FB or EFB? I'm not entirely sure what problem/issue this commit solves